{"uuid": "11cc04c7-e1e0-4da8-8466-11719501bfb0", "counter": 38, "stdout": "\u001b[0;31mfatal: [demo03]: FAILED! => {\"changed\": false, \"msg\": \"Unable to start service vsftpd: Job for vsftpd.service failed because the control process exited with error code.\\nSee \\\"systemctl status vsftpd.service\\\" and \\\"journalctl -xeu vsftpd.service\\\" for details.\\n\"}\u001b[0m\r\n\u001b[0;36m...ignoring\u001b[0m", "start_line": 58, "end_line": 60, "runner_ident": "e53d9eea-f296-4c8a-9973-4dad09d06c8e", "event": "runner_on_failed", "pid": 160190, "created": "2025-05-15T17:37:22.850393+00:00", "parent_uuid": "005056b7-f67d-09ed-2114-00000000000e", "event_data": {"playbook": "/tmp/tmp2mwoc752.yml", "playbook_uuid": "a146ccbe-e0ed-431a-9577-a86734c4c79b", "play": "Corregir configuraci\u00f3n SSL de VSFTPD (Versi\u00f3n 2)", "play_uuid": "005056b7-f67d-09ed-2114-000000000002", "play_pattern": "demo03", "task": "Reiniciar VSFTPD con configuraci\u00f3n b\u00e1sica (sin SSL)", "task_uuid": "005056b7-f67d-09ed-2114-00000000000e", "task_action": "service", "resolved_action": "ansible.builtin.service", "task_args": "", "task_path": "/tmp/tmp2mwoc752.yml:127", "host": "demo03", "remote_addr": "demo03", "res": {"msg": "Unable to start service vsftpd: Job for vsftpd.service failed because the control process exited with error code.\nSee \"systemctl status vsftpd.service\" and \"journalctl -xeu vsftpd.service\" for details.\n", "invocation": {"module_args": {"name": "vsftpd", "state": "restarted", "enabled": true, "daemon_reload": false, "daemon_reexec": false, "scope": "system", "no_block": false, "force": null, "masked": null}}, "_ansible_no_log": false, "changed": false}, "start": "2025-05-15T17:37:18.806374+00:00", "end": "2025-05-15T17:37:22.849954+00:00", "duration": 4.04358, "ignore_errors": true, "event_loop": null, "uuid": "11cc04c7-e1e0-4da8-8466-11719501bfb0"}}